libraries/AP_HAL/board/linux.h

```diff
@@ -227,6 +227,7 @@
 #define HAL_BARO_DEFAULT HAL_BARO_MS5611_SPI
 #define HAL_BARO_MS5611_NAME "ms5611"
 #define HAL_COMPASS_DEFAULT HAL_COMPASS_BBBMINI
+#define HAL_AIRSPEED_MS4515DO_I2C_BUS 2
```

Comment:
we should remove those macros I think.
for the list of buses, we really need a hal.i2c->get_bus_list() function then use that everywhere instead of the current approach.